src/main/java/mate/academy/onlinebookstore/controller/CategoryController.java

```diff
@@ -26,7 +26,7 @@
 import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
-@Tag(name = "Book management", description = "Endpoints for managing product")
+@Tag(name = "Category management", description = "Endpoints for managing product")
```

Comment:
it seems like description is copy-paste in this and OrderController. Check and fix it